boringtun/src/device/mod.rs

```diff
@@ -191,14 +193,36 @@ impl DeviceHandle {
 
         let interface_lock = Arc::new(Lock::new(wg_interface));
 
-        let mut threads = vec![];
+        #[cfg(any(target_os = "macos", target_os = "ios", target_os = "tvos"))]
+        let threads = {
+            let group = dispatch::Group::create();
+            let mut queues = vec![];
+            for i in 0..n_threads {
+                queues.push({
+                    let dev = Arc::clone(&interface_lock);
+                    let group_clone = group.clone();
+                    let queue = dispatch::Queue::global(dispatch::QueuePriority::High);
+                    queue.exec_async(move || {
+                        group_clone.enter();
+                        DeviceHandle::event_loop(i, &dev)
+                    });
+                    queue
+                });
+            }
+            (group, queues)
```

Comment:
Isn't `n_threads` fixed to be `1` in Telio?

Reply:
Yeah, just made it possible here to create multiple queues (if there will be a need for that, mimicking the thread creation), but at this stage, we'll fix it to 1 from `libtelio` side.